definitions/video-roku_system/definition.yml

```diff
@@ -0,0 +1,21 @@
+domain: VIDEO
+# Represents all instances of a customer's Roku App monitored with New Relic's Roku agent.
```

Comment:
Definition description is the same for both VIDEO-ROKU_SYSTEM and VIDEO-ROKU_VIDEO, is that correct?

Reply:
thanks, good callout, I will edit one of them